app/models/user.rb

```diff
@@ -84,31 +82,10 @@ class User < ActiveRecord::Base
     end
   }
 
-  def after_2sv_go_live?
-    Time.zone.now >= TWO_STEP_GO_LIVE_DATE
-  end
-
   def prompt_for_2sv?
     return false if has_2sv?
 
-    if require_2sv?
-      return true if after_2sv_go_live?
-
-      if deferred_2sv_at?
-        last_prompted_ago = Time.zone.now - deferred_2sv_at
-        last_prompted_ago > TWO_STEP_PROMPT_THRESHOLD_PERIOD
-      else
-        true
-      end
-    end
-  end
-
-  def defer_two_step_verification
-    transaction do
-      touch(:deferred_2sv_at)
-
-      EventLog.record_event(self, EventLog::TWO_STEP_PROMPT_DEFERRED)
-    end
+    require_2sv?
```

Comment:
Could just be `require_2sv? && !has_2sv?`

I don't know if that's more or less readable.